backend/app/exporters/serializers/mods.rb

```diff
@@ -143,4 +161,49 @@ def serialize_note(note, xml)
       xml.text note.content
     }
   end
+
+  private
+
+  def process_date_string(date)
+    # use date expression, if present
+    unless date['expression'].nil? || date['expression'].empty?
+      return date['expression']
+    end
+
+    # if end date specified, use begin - end form
+    unless date['end'].nil? || date['end'].empty?
+      return "#{date['begin']} - #{date['end']}"
+
+    # otherwise, just use begin date
+    else
+      return date['begin']
+    end
+  end
+
+  def process_date_attrs(date)
+    attrs = {}
+
+    # qualifier
+    if date.has_key?('certainty')
+      case date['certainty']
+      when "approximate"
+        attrs["qualifier"] = "approximate"
+      when "inferred"
+        attrs["qualifier"] = "inferred"
+      when "questionable"
+        attrs["qualifier"] = "questionable"
+      end
+    end
+    # encoding
+    attrs['encoding'] = "w3cdtf"
+
+    # keyDate
+    attrs['keyDate'] = "yes"
+
+    # point
+    attrs['point'] = "end"
```

Comment:
@avatar382 I believe you could also have a point attribute = 'begin' as well. See comment above.